client/solve.go

```diff
@@ -364,13 +364,11 @@ func parseCacheOptions(opt SolveOpt) (*cacheOptions, error) {
 		cacheExports []*controlapi.CacheOptionsEntry
 		cacheImports []*controlapi.CacheOptionsEntry
 		// legacy API is used for registry caches, because the daemon might not support the new API
-		legacyExportRef  string
 		legacyImportRefs []string
```

Comment:
legacyImportRefs is no longer referred as well?

Reply:
I guess not, since I need to build the "cache-from" attr based on the those legacyImportRefs to compatible with the legacy frontend.

Reply:
pls see client/solve.go:437 in this change.